sources/api/migration/migration-helpers/src/common_migrations.rs

```diff
@@ -739,3 +739,106 @@ impl Migration for ReplaceTemplateMigration {
         Ok(input)
     }
 }
+
+// =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+
+/// We use this migration when we add metadata and want to make sure they're removed before we go
+/// back to old versions that don't understand them.
+#[derive(Debug)]
+pub struct SettingsMetadata<'a> {
+    pub setting: &'static str,
+    pub metadata: &'a [&'static str],
```

Comment:
This list is always going to be static in the code, too, so I think we can avoid the `'a` lifetimes and make it `'static` as well.

Reply:
Similarly, I think the `Vec` in `AddMetadataMigration` could be a `&'static []`, just like this, for the same reason?